### PR TITLE
added Juniper EX2200 module for Junos 15 and above

### PR DESCRIPTION
**Pull request by julsemaan**
# Description
added Juniper EX2200 module for Junos 15 and above

# Impacts
Switch modules

# Delete branch after merge
YES

# Checklist
(REQUIRED) - [yes, no or n/a]
- [ ] Document the feature
- [ ] Add unit tests
- [ ] Add acceptance tests (TestLink)

# NEWS file entries
(REQUIRED, but may be optional...)
## Enhancements
* Improved support for Juniper switches running Junos 15 and above

### CODE REVIEW

File: lib/pf/Switch/Juniper/EX2200_v15.pm [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+}
+
+
+=head1 AUTHOR
+
+Inverse inc. <info@inverse.ca>
+
+=head1 COPYRIGHT
+
+Copyright (C) 2005-2018 Inverse inc.
```
Comment (fdurand) [inline lib/pf/Switch/Juniper/EX2200_v15.pm]: 2019

**Comment by fdurand** [comment]
It's just because of 'Calling-Station-Id' => $mac, ?

**Comment by julsemaan** [comment]
Looks like it, did this a while ago at a customer and brought back the changes here during the upgrade

**Comment by fdurand** [comment]
ok , so just fix the date and i will merge it. Then you will need to merge one of my PR ;-)

**Comment by julsemaan** [comment]
Date is fixed

**Pull request merged**